src/restaurants/details/DetailPage.tsx

```diff
@@ -18,57 +18,47 @@ const DetailPage = () => {
 	const { setIsError } = useContext(HeaderContext);
 	let { name } = useParams<{ name: string }>();
 
-	const {
-		data: restaurantData,
-		isLoading: isRestaurantLoading,
-		error: restaurantError,
-		isError: restaurantIsError
-	} = getMenuPerRestaurantQuery(name!);
 	const {
 		data: menuData,
```

Comment:
We should use meaningul names, in this case menuData could be misleading since the data holds information about the restaurant, not only the menu, I'd recommend to change this for 'restaurant'